### PR TITLE
Improve error messages for registerWorker with invalid inputs

### PR DESCRIPTION
**Pull request by djmitche**
This came up in slack today as a 500 and I figured we could do better.

I changed 'Token validation' since there's no token being validated, so it's kind of a misleading error message.

### CODE REVIEW

**Comment by djmitche** [comment]
https://console.cloud.google.com/errors/CM2Ol4O1uNyw8wE?time=P30D&project=heroku-logging

File: changelog/V9gQQys8RWiQUxYiQwsQSg.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+level: silent
```
Comment (owlishDeveloper) [inline changelog/V9gQQys8RWiQUxYiQwsQSg.md]: why "silent" and no description?

File: services/worker-manager/src/providers/aws.js [hunk 2]
```diff
@@ -196,8 +196,8 @@ class AwsProvider extends Provider {
     }
 
     const {document, signature} = workerIdentityProof;
-    if (!document || !signature) {
-      throw new ApiError('Token validation error');
+    if (!document || !signature || !(typeof document === "string")) {
+      throw new ApiError('Request must include both a document (string) and a signature');
```
Comment (owlishDeveloper) [inline services/worker-manager/src/providers/aws.js]: I thought we wanted to keep the error messages vague on purpose? In case of a malicious action

Comment (djmitche) [reply services/worker-manager/src/providers/aws.js]: The vagueness was about the validity of the document -- this is just about adherence to the expected form of the request, which is public information anyway.

File: changelog/V9gQQys8RWiQUxYiQwsQSg.md (hunk 1 above)
Comment (djmitche) [reply changelog/V9gQQys8RWiQUxYiQwsQSg.md]: "silent: Minor bugfixes where the bugs did not impact many users"

**Pull request merged**